js/ui/data_grid/ui.data_grid.export.js

```diff
@@ -781,13 +743,12 @@ dataGridCore.registerModule('export', {
                     const items = [];
 
                     formats.forEach((formatType) => {
-                        let exportMethod = 'exportTo';
+                        const exportMethod = 'exportTo';
```

Comment:
let's remove this and change

```
this._exportController[exportMethod]
```

to 

```
this._exportController.exportTo
```

below in code

Reply:
[fixed](https://github.com/DevExpress/DevExtreme/pull/23643/commits/318a8ef73a88264d78c65dbe2c25a6e7c5ea541d)